src/content/basics/giant-swarm-platform-access/index.md

```diff
@@ -0,0 +1,71 @@
++++
+title = "The Giant Swarm Platform Access Routes"
+description = "General Giant Swarm Platform Access Routes Overview"
+date = "2019-05-08"
+weight = 20
+type = "page"
+categories = ["basics"]
++++
+
+# Giant Swarm Platform Access Routes
+
+## Intro
+Access to Giant Swarm platforms can be split into two parts. 
+
+1. Admin Access - designated for the Giant Swarm employees for managing/development/support purposes.
+
+2. User Access - designated for GiantSwarm customers to interact with the offered services.
+
+Both are managed via Giant Swarm VPN hopp that enables connection through jumphost of the cluster.
```

Comment:
What is a `hopp `?

Reply:
My understand so far is that VPN protection of the Giant Swarm API is only optional. We have some installations where the API is reachable from everywhere.

Reply:
Yes,  GS API protection is optional

Reply:
Anyway, I would try to find a more suitable term. Jump host? Bastion Host? VPN entry point? Server?